build-release-tools/application/generate_manifest.py

```diff
@@ -114,16 +118,16 @@ def main():
             if args.date == "current":
                 utc_now = datetime.utcnow()
                 day_str = utc_now.strftime("%Y%m%d")
-                dest_manifest = "{branch}-{day}".format(branch=slice_branch, day=day_str)
-                generator = ManifestGenerator(dest_manifest, args.branch, args.builddir, git_credential=args.git_credential, jobs=args.jobs, force=args.force)
+                dest_manifest = "{type}-{branch}-{day}".format(type=args.type, branch=slice_branch, day=day_str)
```

Comment:
when if get rid of "--type". the dest_manifest file name can extract the "type" you want from the template file ?

or you can pass the template file "manifset-$type.json"  to SpecifyDayManifestGenerator().